db/migrate/20210103163430_add_index_to_kind_and_service_on_ads.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexToKindAndServiceOnAds < ActiveRecord::Migration[6.1]
+  def change
+    add_index :ads, :kind
+    add_index :ads, :service
```

Comment:
This one was renamed (again) to `category` in master https://github.com/vlado/earthquake-croatia/pull/96. Sorry about that.

Reply:
:) No worries. I was more surprised to see repurposing `kind` into a completely different concept.